### PR TITLE
Fs jul19 yevhen sag - human rights

### PR DESCRIPTION
**Pull request by eugeneSahach**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+<body>
+  <h1>Universal Declaration of Human Rights</h1>
+  <h2>Preamble</h2>
+  <p>Whereas recognition of the inherent dignity and of the equal and inalienable
+    rights of all members of the human family is the foundation of freedom, justice
+    and peace in the world,<br>Whereas disregard and contempt for human rights have resulted in barbarous
+    acts which have outraged the conscience of mankind, and the advent of a world
+    in which human beings shall enjoy freedom of speech and belief and freedom
+    from fear and want has been proclaimed as the highest aspiration of the common
+    people,<br>Whereas it is essential, if man is not to be compelled to have recourse, as a last
```
Comment (gestap) [inline src/index.html]:
```suggestion
    people,<br>
    Whereas it is essential, if man is not to be compelled to have recourse, as a last
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    resort, to rebellion against tyranny and oppression, that human rights should be
+    protected by the rule of law,<br>Whereas it is essential to promote the development of friendly relations between
+    nations,<br>Whereas the peoples of the United Nations have in the Charter reaffirmed their
```
Comment (gestap) [inline src/index.html]:
```suggestion
    nations,<br>
    Whereas the peoples of the United Nations have in the Charter reaffirmed their
```

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    faith in fundamental human rights, in the dignity and worth of the human person
+    and in the equal rights of men and women and have determined to promote
+    social progress and better standards of life in larger freedom,<br>Whereas Member States have pledged themselves to achieve, in cooperation
+    with the United Nations, the promotion of universal respect for and observance of
+    human rights and fundamental freedoms,<br>Whereas a common understanding of these rights and freedoms is of the
+    greatest importance for the full realization of this pledge,<br>Now, therefore,<br>The General Assembly,<br>Proclaims this Universal Declaration of Human Rights as a common standard of
```
Comment (gestap) [inline src/index.html]:
```suggestion
    greatest importance for the full realization of this pledge,<br>
    Now, therefore,<br>
    The General Assembly,<br>
    Proclaims this Universal Declaration of Human Rights as a common standard of
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  margin-bottom: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+#link {
```
Comment (gestap) [inline src/style.css]: do not use `id` for style elements, use `class` for this task

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+  margin-bottom: 0;
```
Comment (gestap) [inline src/style.css]: try to write margin in one line, it will be better

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+h2 {
```
Comment (gestap) [inline src/style.css]: do not style tag, create a class and style it

**Comment by yevhen-sahach** [comment]
Switched referencing from tags and id to classes, also made little changes in first paragraph so it's more readable.

**Review by YuriiHoliuk — APPROVED** [review]

**Pull request closed without merging**